### PR TITLE
Replace MITRE in project statement "maintainers" section and package.json

### PR DESCRIPTION
**Pull request by mint-thompson**
Indicate that project is maintained by the HL7 community. This also applies to the "description" and "author" fields in `package.json`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "vscode-language-fsh",
   "displayName": "FHIR Shorthand",
-  "description": "FHIR Shorthand (FSH) Language Support by MITRE",
+  "description": "FHIR Shorthand (FSH) Language Support by HL7 community",
```
Comment (cmoesel) [inline package.json:4]: I think we can/should just say HL7 here:
```suggestion
  "description": "FHIR Shorthand (FSH) Language Support by HL7",
```

File: package.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
   "version": "1.16.1",
-  "author": "The MITRE Corporation",
+  "author": "HL7 community",
```
Comment (cmoesel) [inline package.json:6]: Here I'd also just refer to HL7 as the author:
```suggestion
  "author": "Health Level Seven International",
```

**Review by cmoesel — CHANGES_REQUESTED** [review]
I've suggested a few small tweaks.

Also, can you add a `# LICENSE` section to the README like we have in many of the other repositories? It should include a copyright statement as well. We want to make it very clear that the copyright was transferred.

**Review by cmoesel — APPROVED** [review]
Looks good to me. Thank you!

**Review by jafeltra — APPROVED** [review]

**Pull request merged**